tests/integration/api_v3/rpc/dposUnlocks.test.js

```diff
@@ -0,0 +1,154 @@
+			expect(result.meta).toMap(metaSchema);
+		}
+	});
+
+	it('Returns unlocks when requested for existing account by publicKey', async () => {
+		if (refTransaction.sender.publicKey) {
+			const response = await getUnlocks({ publicKey: refTransaction.sender.publicKey });
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toMap(unlockSchema);
+			expect(result.data.unlocking.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.unlocking.length).toBeLessThanOrEqual(10);
+			expect(result.meta).toMap(metaSchema);
+		}
+	});
+
+	it('Returns unlocks when requested for existing account by address and limit=5', async () => {
+		const response = await getUnlocks({
+			address: refTransaction.sender.address,
+			limit: 5,
+		});
+		expect(response).toMap(jsonRpcEnvelopeSchema);
+		const { result } = response;
+		expect(result.data).toMap(unlockSchema);
+		expect(result.data.unlocking.length).toBeGreaterThanOrEqual(1);
+		expect(result.data.unlocking.length).toBeLessThanOrEqual(5);
+		expect(result.meta).toMap(metaSchema);
+	});
+
+	it('Returns unlocks when requested for existing account by address, limit=5 and offset=1', async () => {
+		const response = await getUnlocks({
+			address: refTransaction.sender.address,
+			limit: 5,
+			offset: 1,
+		});
+		expect(response).toMap(jsonRpcEnvelopeSchema);
+		const { result } = response;
+		expect(result.data).toMap(unlockSchema);
+		expect(result.data.unlocking.length).toBeGreaterThanOrEqual(1);
+		expect(result.data.unlocking.length).toBeLessThanOrEqual(5);
+		expect(result.meta).toMap(metaSchema);
+	});
+
+	it('Returns unlocks when requested for existing account by address, name and limit=5', async () => {
+		if (refTransaction.sender.name) {
+			const response = await getUnlocks({
+				address: refTransaction.sender.address,
+				name: refTransaction.sender.name,
+				limit: 5,
+			});
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toMap(unlockSchema);
+			expect(result.data.unlocking.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.unlocking.length).toBeLessThanOrEqual(5);
+			expect(result.meta).toMap(metaSchema);
+		}
+	});
+
+	it('Returns unlocks when requested for existing account by address, publicKey and limit=5', async () => {
+		if (refTransaction.sender.publicKey) {
+			const response = await getUnlocks({
+				address: refTransaction.sender.address,
+				publicKey: refTransaction.sender.publicKey,
+				limit: 5,
+			});
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result.data).toMap(unlockSchema);
+			expect(result.data.unlocking.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.unlocking.length).toBeLessThanOrEqual(5);
+			expect(result.meta).toMap(metaSchema);
+		}
+	});
+
+	it('No Params -> invalid param', async () => {
+		const response = await getUnlocks();
```

Comment:
Do we not require the default params object?
```suggestion
		const response = await getUnlocks({});
```